Automattic/sensei
# Update footer design for graded quiz

Resolves https://github.com/Automattic/sensei/issues/7116

## Proposed Changes
* Update footer design to match the new design
* Fixed extra lateral spacing issue in footer in Divi, Twenty Twenty Three and some other themes

## Testing Instructions


1. Create a course with some lessons with quizzes
2. Apply different quiz settings on different quizzes, for example, mark one as pass required, one with auto grade enabled, one with reset enabled, and different combinations of these. The last lesson should have a quiz to check if the next lesson button renders unintentionally when there is no more lesson after that.
3. Take those quizzes as a student. keep some of the quizzes incomplete, some awaiting grade, some passed, some failed, some just graded
4. Check that the style holds in different scenarios
5. Check for mobile view
6. Check for different themes
7. Check the lesson view as well to make sure nothing is broken there
8. Check with the learning mode turned off as well to make sure nothing is broken there.

Passed:
<img width="1154" alt="Screenshot 2023-10-03 at 6 38 45 PM" src="https://github.com/Automattic/sensei/assets/6820724/d992d96b-208a-482c-86dd-8ed03bbdba12">

Failed with reset:
<img width="1154" alt="Screenshot 2023-10-03 at 6 40 40 PM" src="https://github.com/Automattic/sensei/assets/6820724/a40bd658-3773-401d-8589-7c0a9e767d0f">

Last lesson passed with no reset:
<img width="1154" alt="Screenshot 2023-10-03 at 6 41 27 PM" src="https://github.com/Automattic/sensei/assets/6820724/efd8ef5e-612b-4e57-9af4-60eaeb31fb28">


## Pre-Merge Checklist
